app/constants/view.ts

```diff
@@ -16,19 +16,25 @@ export const TABLET_SIDEBAR_WIDTH = 320;
 export const IOS_STATUS_BAR_HEIGHT = 20;
 export const IOS_DEFAULT_HEADER_HEIGHT = 44;
 export const ANDROID_DEFAULT_HEADER_HEIGHT = 56;
-export const LARGE_HEADER_TITLE = 60;
-export const HEADER_WITH_SEARCH_HEIGHT = -16;
+export const LARGE_HEADER_TITLE_HEIGHT = 60;
 export const HEADER_WITH_SUBTITLE = 24;
 export const KEYBOARD_TRACKING_OFFSET = 72;
-export const HEADER_SEARCH_HEIGHT = SEARCH_INPUT_HEIGHT + 5;
-export const HEADER_SEARCH_BOTTOM_MARGIN = 10;
+export const UNLOCKED_SEARCH_BOTTOM_MARGIN = Platform.select({android: -14, default: -14});
+export const LOCKED_SEARCH_BOTTOM_MARGIN = Platform.select({android: 11, default: 0});
+
+export const TABLET_SEARCH_HEIGHT_COLLAPSED = Platform.select({android: 88, default: 98}); // 98 total, insets=24, 74
+export const TABLET_SEARCH_HEIGHT_EXPANDED = Platform.select({android: 145, default: 182}); // 182 total, insets=24, 158
```

Comment:
can you explain this values?

Reply:
These are the values for the expanded and collapsed heights.  I'll fix these and account for the component heights needed to get the values in stead of hard coding them.  I'll get those fixed and ask for a re-review.